.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# Checklist for submitter
+
+If some of the following don't apply, please write a short explanation why.
+
+- [ ] Changes file added (if needed)
+- [ ] Documented any API changes
+- [ ] Added tests for all functionality
```

Comment:
```suggestion
- [ ] Tests and/or manual QA for all functionality
```

Reply:
How about we keep these as two separate items instead? I would rather not convey the message that not writing tests is ok, as long as you use it. Because it's very likely that you're missing things in your tests by hand.